src/index.html

```diff
@@ -4,9 +4,486 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="header catalog__header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="Moyo_logo">
+      </a>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <a
+            href="#apple"
+            class="header__link is-active"
+            id="apple"
+          >
+            <li>
+              Apple
+            </li>
+          </a>
+          <a href="#samsung" class="header__link">
+            <li>
+              Samsung
+            </li>
```

Comment:
Better create `nav` block and ul and li, a would be a nav block elements. 
The correct usage `li` tag - wrap `a` tag inside li.
```suggestion
      <nav class="header__nav nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#Apple" class="nav__link is-active">Apple</a>
          </li>
          <li class="nav__item">
            <a href="#Samsung" class="nav__link">Samsung</a>
          </li>
          <li class="nav__item">
            <a href="#Phones" class="nav__link">Смартфоны и телефоны</a>
          </li>
```